devito/symbolics/manipulation.py

```diff
@@ -344,3 +344,14 @@ def indexify(expr):
         except AttributeError:
             pass
     return expr.xreplace(mapper)
+
+
+def aligned_indices(i, j, spacing):
+    """
+    Check if two indices are aligned. Two indices are aligned if they
+    differ by an Integer*spacing
```

Comment:
full stop

Reply:
probably nitpicking
I'm actually tempted to say that if you're using this function only in `is_on_grid`, since I really doubt it's ever going to be used elsewhere, you may as well drop it and move the content directly in that method

Reply:
I prefer it explicit and separated